src/main/webapp/WEB-INF/view/navbar.jsp

```diff
@@ -1,5 +1,6 @@
 <nav>
   <% String navBarUser = (String)request.getSession().getAttribute("user"); %>
+  <% List<String> adminList = (List<String>) request.getSession().getAttribute("adminList"); %>
```

Comment:
Why do we want the list here?

Reply:
I was attempting to add the admin page as a tab only when the user is an admin, for some reason when I called adminList.contains(navBarUser) it would give me an error. I forgot to delete it from my attempts.

Reply:
I see. We can try adding `boolean isAdmin` in the `User` entity then adding a condition to check for this.

Reply:
I do have an isAdmin boolean currently in the User. How would I go about using it in the navbar.jsp?

Reply:
Sorry I missed that. You can use `UserStore` to get the `User` object with `navBarUser` as the parameter, then call on `isAdmin()`.